docs/docs/7-migration-guide/migration-from-v7-to-v8.md

```diff
@@ -479,6 +479,76 @@ It is recommended to use the `AutocompleteField` or the `SelectField` components
 
 ## ESLint
 
+### ESLint upgrade from v8 to v9 with ESM
+
+Update ESLint to v9
+
+`package.json`
+
+```diff
+- "eslint": "^8.0.0",
++ "eslint": "^9.0.0",
+```
+
+An ESM compatible ESLint config is required. Delete the related `.eslintrc.json` and move the configured rules to the new ESLint flat configuration `eslint.config.mjs`.
+
+Migration Guide of ESLint 9.0 can be found here: [Migration Guide](https://eslint.org/docs/latest/use/migrate-to-9.0.0)
+
+#### `admin/eslint.config.mjs`
```

Comment:
Should we link to the files in Demo instead? The code here may become out of date fast.

Reply:
We can, but i thought it could be missleading, because we are disabling some rules (e.g. translation is not required in demo) for demo, what we normally want to have enabled in projects